astrowidgets/core.py

```diff
@@ -500,12 +524,13 @@ def get_markers(self, x_colname='x', y_colname='y',
                 [xy_col[:, 0], xy_col[:, 1], sky_col],
                 names=(x_colname, y_colname, skycoord_colname))
         else:
-            markers_table = Table(xy_col.T, names=(x_colname, y_colname))
```

Comment:
Why is `.T` removed? Was it a bug? Has the API changed? I remember adding `.T` but I don't remember why.

Reply:
It was a bug...the `.T` raised an error.

Reply:
Weird... Pretty sure it worked when I tested it a while ago. Ah, well. :woman_shrugging: